### DISCUSSION TITLE
Credit for customers

### DISCUSSION THREAD

**Discussion by BaxAndrei** [crater-invoice-inc/crater#580] [Ideas]
**Is your feature request related to a problem? Please describe.**
To be able to add credit for customers that you can use in future invoices. For example, if the client has an invoice of 100 euros and pays 200, you can add the remaining 100 as credit for the future.

**Describe the solution you'd like**
A new section like the one with payments in which you can add credits exactly as you do with payments. Then you have the option to use the customer's credit for a payment.

**Describe alternatives you've considered**
-

**Comment by b4allb4u** [comment]
If possible, please add an option of credit limit to the customers at the time of customer creation.

**B4ALLB4U**
